back/app.yaml

```diff
@@ -24,3 +24,7 @@ handlers:
   secure: always
   static_files: react-build/index.html
   upload: react-build/index.html
+- url: /(.*\.js)$
```

Comment:
Can you switch this block with the fallback handler above? Otherwise I think it's never reached

Reply:
@pylipp I have changed the route with a more secure one that follows a more specific path